src/utils/api/block/index.js

```diff
@@ -1,5 +1,19 @@
+import { subscribe, unsubscribe } from '../ws';
+
+const WS_ENDPOINTS = {
+  BLOCKS_CHANGE: 'blocks/change',
+};
```

Comment:
I'm not a fan of UPPERCASE constants. we have the `const`  variables now.  I asked to changed it in your other PR, I'm open to uppercase or camelcase, but we should decide on one, and use it consistently through out the application.